internal/host/installcmd.go

```diff
@@ -71,6 +71,12 @@ func (i *installCmd) GetStep(ctx context.Context, host *models.Host) (*models.St
 		data["HOST_NAME"] = hostname
 	}
 
+	cmdArgsTmpl = cmdArgsTmpl + " || ( returnCode=$?; podman run --rm --privileged " +
+		"-v /run/systemd/journal/socket:/run/systemd/journal/socket -v /var/log:/var/log " +
```

Comment:
I think this logic should be added the agent command execution.
Meaning:
Step will contain a ReportLogs option with all relevant fields.
When it's set the agent will wrap the command with this logic.